src/assets/scss/app.scss

```
@@ -0,0 +1,19 @@
+@import "~bulma";
+@import "~bulma-timeline";
+
+.navbar {
+  border-bottom: 1px solid #eee;
+  margin-bottom: 8px;
+}
+
+.gh-username img{
```

```suggestion
.gh-username img {
```

space before the curly brace